src/tui/editor.zig

```diff
@@ -5930,6 +5941,15 @@ pub const Editor = struct {
         self.completion_is_complete = is_incomplete;
     }
 
+    pub fn add_document_symbol_done(self: *Self, _: tp.message) Result {
```

Comment:
Symbols are likely not just file local and probably can be handled entirely in `mainview`. I'm not entirely sure about this though, just guessing.